src/js/components/releases/releases.js

```diff
@@ -0,0 +1,314 @@
+import { useDebounce } from '../../utils/debouncehook';
+import { useLocationParams } from '../../utils/liststatehook';
+import { getOnboardingComponentFor } from '../../utils/onboardingmanager';
+import ChipSelect from '../common/chipselect';
+import InfoHint from '../common/info-hint';
+import Search from '../common/search';
+import AddArtifactDialog from './dialogs/addartifact';
+import ReleaseDetails from './releasedetails';
+import ReleasesList from './releaseslist';
+
+const refreshArtifactsLength = 60000;
+
+const tabs = [
+  { key: 'releases', title: 'Releases', component: ReleasesList, isApplicable: () => true },
+  {
+    key: 'delta',
+    title: 'Delta Artifacts generation',
+    component: 'div',
+    isApplicable: ({ features: { hasDeltaProgress }, tenantCapabilities: { canDelta } }) => hasDeltaProgress && canDelta
+  }
+];
+
+const UploadArtifactOnboardingComponent = ({ dropzoneRef, onboardingState, demoArtifactLink, releases }) => {
+  if (!dropzoneRef.current || releases.length) {
+    return null;
+  }
+  const dropzoneAnchor = { left: 30, top: dropzoneRef.current.offsetTop + dropzoneRef.current.offsetHeight };
+  return getOnboardingComponentFor(
+    onboardingSteps.UPLOAD_PREPARED_ARTIFACT_TIP,
+    { ...onboardingState, demoArtifactLink },
+    { anchor: dropzoneAnchor, place: 'left' }
+  );
+};
+
+const EmptyState = ({ canUpload, dropzoneRef, uploading, onDrop, onUpload }) => (
+  <div className="dashboard-placeholder fadeIn" style={{ fontSize: '16px', margin: '8vh auto' }} ref={dropzoneRef}>
+    <Dropzone activeClassName="active" disabled={uploading} multiple={false} noClick={true} onDrop={onDrop} rejectClassName="active">
+      {({ getRootProps, getInputProps }) => (
+        <div {...getRootProps({ className: uploading ? 'dropzone disabled muted' : 'dropzone' })} onClick={() => onUpload()}>
+          <input {...getInputProps()} disabled={uploading} />
+          <p>
+            There are no Releases yet.{' '}
+            {canUpload && (
+              <>
+                <a>Upload an Artifact</a> to create a new Release
+              </>
+            )}
+          </p>
+        </div>
+      )}
+    </Dropzone>
+  </div>
+);
+
+const Header = ({ canUpload, existingTags = [], features, releasesListState, setReleasesListState, onUploadClick, tenantCapabilities, uploadButtonRef }) => {
+  const { hasReleaseTags } = features;
+  const { selectedTags = [], searchTerm, searchTotal, tab = tabs[0].key, total } = releasesListState;
+
+  const searchUpdated = searchTerm => setReleasesListState({ searchTerm });
+
+  const onTabChanged = (e, tab) => setReleasesListState({ tab });
+
+  const onTagSelectionChanged = ({ selection }) => setReleasesListState({ selectedTags: selection });
+
+  const availableTabs = useMemo(
+    () =>
+      tabs.reduce((accu, tab) => {
+        if (tab.isApplicable({ features, tenantCapabilities })) {
+          accu.push(tab);
+        } else {
+          accu.push({ ...tab, disabled: true });
```

Comment:
Just a confirmation: if the tab is not supported, we render it as "disabled" (click and nothing happens". Right?

Reply:
yes... but to give a bit more context we could show a tooltip and track any clicks on it to gauge interest 🙃
(same goes with the tag functionality that's hidden behind the `$HAVE_RELEASE_TAGS`)

Reply:
I think it would be better to allow the user to click it, but show placeholder text - similar to the empty states we have elsewhere. I should have specified in the figma files, sorry! I can take care of it. It shouldn't block this PR though